examples/prometheus/slo_rules.yaml

```diff
@@ -0,0 +1,58 @@
+rules:
+
+  # HTTP requests availability rules
+  - metadata_matcher:
+      - key: event_type
+        operator: isEqualTo
+        value: http_request_result
+    failure_conditions:
+      # Requests to api are failed events if resulted with status code higher or equal to 500
+      - key: code
+        operator: numberIsEqualOrHigherThan
+        value: 500
+    additional_metadata:
+      slo_type: api_availability
+      slo_version: 6
```

Comment:
I would not use multiple `slo_version` values here. It may be misleading.

Reply:
Yeah, those should be same everywhere here I just missed some of them, thanks